apis/r/R/TileDBGroup.R

```diff
@@ -32,10 +32,15 @@ TileDBGroup <- R6::R6Class(
                    "factory method as e.g. 'SOMACollectionCreate()'."), call. = FALSE)
       }
 
-      spdl::info("Creating new {} at '{}'", self$class(), self$uri)
-      tiledb::tiledb_group_create(uri = self$uri, ctx = private$.tiledb_ctx)
+      spdl::debug("[TileDBGroup$create] Creating new {} at '{}' at {}",
+                  self$class(), self$uri, self$tiledb_timestamp)
 
-      self$open("WRITE", internal_use_only = "allowed_use")
+      #tiledb::tiledb_group_create(uri = self$uri, ctx = private$.tiledb_ctx)
+      private$.soma_context <- soma_context()  # FIXME via factory and paramater_config
+      c_group_create(self$uri, self$class(), private$.soma_context,
+                     self$.tiledb_timestamp_range)  ## FIXME: use to be added accessor
+
+      #self$open("WRITE", internal_use_only = "allowed_use") # FIXME
```

Comment:
Can we clean out the commented code?
```suggestion
      private$.soma_context <- soma_context()  # FIXME via factory and paramater_config
      c_group_create(self$uri, self$class(), private$.soma_context,
                     self$.tiledb_timestamp_range)  ## FIXME: use to be added accessor
```